sqlalchemy_ibmi/backend.py

```diff
@@ -88,15 +106,47 @@ def create_connect_args(self, url):
         return [[";".join(connectors)], connect_args]
 
 
-class AS400Dialect_pyodbc(PyODBCConnector, DB2Dialect):
+class AS400DialectPyodbc(PyODBCConnector, DB2Dialect):
+    """ IBM i pyodbc sqlalchemy dialect """
+
+    def get_temp_table_names(self, connection, schema=None, **kw):
+        pass
+
+    def get_temp_view_names(self, connection, schema=None, **kw):
+        pass
+
+    def get_check_constraints(self, connection, table_name, schema=None, **kw):
+        pass
+
+    def get_table_comment(self, connection, table_name, schema=None, **kw):
+        pass
+
+    def do_begin_twophase(self, connection, xid):
+        pass
+
+    def do_prepare_twophase(self, connection, xid):
+        pass
+
+    def do_rollback_twophase(self, connection, xid, is_prepared=True,
+                             recover=False):
+        pass
+
+    def do_commit_twophase(self, connection, xid, is_prepared=True,
+                           recover=False):
+        pass
+
+    def do_recover_twophase(self, connection):
+        pass
+
+    def get_isolation_level(self, dbapi_conn):
+        pass
+
     supports_unicode_statements = True
     supports_sane_rowcount = False
     supports_sane_multi_rowcount = False
-    supports_native_decimal = True
-    supports_char_length = True
     supports_native_decimal = False
-
-    pyodbc_driver_name = "iSeries Access ODBC Driver"
+    supports_char_length = True
+    pyodbc_driver_name = "IBM i Access ODBC Driver"
```

Comment:
Please do this change in a separate PR. Changing the driver name is not a linter change.